projects/vdk-core/src/vdk/internal/core/errors.py

```diff
@@ -10,6 +10,8 @@
 """
 from __future__ import annotations
 
+from logging import Logger
+
```

Comment:
why this?

Reply:
left over, will remove it

Reply:
Actually, nvm, it's for the type hints. I'm leaving it as is.